charts/pega/README.md

```diff
@@ -758,40 +758,61 @@ pegasearch:
 ```
 ## Deploying Pega with external kafka for stream
 
-
+Deployment of stream with external kafka requires Pega Infinity 8.4 or later.
```

Comment:
Helm supports the deployment of various Pega Infinity versions. Pega Infinity supports external kafka since 8.4.
Since helm supports previous Infinity versions as well, the above comment is added to mention this requirement

Reply:
but the context here, I feel, is mis-leading...it wasn't "external kafka" but a stream tier that connected to a Kafka service on those ports...it was still embedded. How about we treat it this way?
Pega Platform deployments support embedded Kafka beginning in Pega Infinity 8.4; for release 8.8 and later, Pega has deprecated support for using embedded Kafka using stream nodes. For 8.8, new deployment should use the newly supported Externalized Kafka service to experience an improved stability and configuration flexibility.

Reply:
we don't have a stream tier when external Kafka is enabled. Secondly, Embedded Kafka is supported since 7.4.
As I mentioned earlier, a customer may deploy with Pega Infinity 8.3 version, in this case, they should not enable external Kafka because it is not supported. Probably we can discuss it further in a call.

Reply:
Pega Platform deployments support externalize Kafka beginning in Pega Infinity 8.4; for release 8.8 and later, Pega has deprecated support for using embedded Kafka using stream nodes. For 8.8, new deployment should use the newly supported Externalized Kafka service configurations to experience an improved stability and configuration flexibility.

Reply:
now that I've looked the pega readme with you changes, can we restructure just a bit??
Deployment of stream with external kafka requires Pega Infinity 8.4 or later. Starting from 8.7 Pega deprecated the use of embedded stream (Stream tier or “Stream” nodetype). 
For new deployments, Pega recommends deploying Pega Platform using an external Kafka configuration as a stream service provider to use your own managed Kafka infrastructure. Pega provides parameters for this configuration in the Pega Helm chart so your Kafka services start when your Pega nodes start. For details and requirements about configuring Kafka, see [Kafka Cluster requirements](https://github.com/pegasystems/pega-helm-charts/blob/e66b5398f346b9c94d8e093ec9decb0b765e6d01/charts/pega/KafkaClusterRequirement.md).
Starting in 8.8, Pega supports migrating existing deployments to use an external Kafka configuration as a stream service provider using the Helm charts. To use your own managed Kafka infrastructure without the use of stream nodes, Pega provides instructions to run a migration with downtime and potential data loss or with minimal downtime and no downtime. For migration steps, see [Switch from embedded Stream to External Kafka](https://github.com/pegasystems/pega-helm-charts/blob/e66b5398f346b9c94d8e093ec9decb0b765e6d01/charts/pega/MigrationToExternalStream.md).